backend/src/main.py

```
@@ -86,6 +86,9 @@ def start_async_loop():
     # Inform services about the background loop
     auto_trader_service.set_loop(background_loop)
 
+    # Run post_init in the background loop
+    background_loop.create_task(auto_trader_service.post_init())
```

Ruff (0.15.14)

[warning] Store a reference to the return value of `background_loop.create_task`

(RUF006)